python/graphstorm/sagemaker/README.md

```diff
@@ -0,0 +1,198 @@
+# Amazon SageMaker Support
+This submodule provides support to run GraphStorm training and inference on Amazon SageMaker.
+To build a SageMaker compatible docker image, please refer to [Build GraphStorm SageMaker docker image] (https://github.com/awslabs/graphstorm/docker/sagemaker).
+
+## Launch SageMaker tasks
+Use scripts under graphstorm.sagemaker.launch to launch SageMaker tasks.
+Please make sure you already setup your SageMaker environments.
+Please refer to [Amazon SageMaker service](https://aws.amazon.com/pm/sagemaker) for how to get access to Amazon SageMaker.
+
+### Launch GraphStorm training/inference using Amazon SageMaker service
+
+#### Launch train task using built-in training script
+
+##### Preparing training data and training task config.
+```
+cd ~/
+git clone https://github.com/awslabs/graphstorm.git
+GS_HOME=~/graphstorm/
+python3 $GS_HOME/tools/partition_graph.py --dataset ogbn-arxiv \
+                                          --filepath /tmp/ogbn-arxiv-nc/ \
+                                          --num_parts 2 \
+                                          --output /tmp/ogbn_arxiv_nc_2p
+```
+
+You need to upload /tmp/ogbn_arxiv_nc_2p into S3. You also need to upload the yaml config file into S3.
+You can find the example yaml file in https://github.com/awslabs/graphstorm/blob/main/training_scripts/gsgnn_np/arxiv_nc.yaml.
```

Comment:
Let's put the instructions before the code so the user know what they're running.